<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gerer_abonneP</class>
 <widget class="QWidget" name="gerer_abonneP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>69</y>
     <width>941</width>
     <height>691</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>390</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>350</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EMAIL">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>390</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Date de naissance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NOM">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="envoyerdemandeabonnement">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DATE">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>300</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Demande d'abonnement Parking</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PRENOM">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="CIN">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="frr">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>300</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="TABDE">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>220</y>
       <width>391</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LES DEMANDES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficher_demande">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>470</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Voir les demandes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer_demande">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>470</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer demande</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>270</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabP">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>230</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>490</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="filtre_abo">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>490</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>190</y>
       <width>239</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="modifier">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="supprimer">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="retourner">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>460</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retourner</string>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>nom</zorder>
    <zorder>prenom</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>tabP</zorder>
    <zorder>save</zorder>
    <zorder>lineEdit</zorder>
    <zorder>filtre_abo</zorder>
    <zorder>retourner</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLineEdit" name="id5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant parking</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de place</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabplace">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>250</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="num5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>390</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouterplaceparking">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>450</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter </string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficherplaceparking">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>510</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifierplaceparking">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>510</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>modiifer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="GESTIONABONNEPARKING">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestion abonne Parking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GESTIONPLACEPARKING">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestion PLACE parking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="consulterdemande">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Consulter les demandes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demandeabonnement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Demande abonnement</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
